.github/workflows/test.yml

```diff
@@ -6,101 +6,77 @@ on:
   - workflow_dispatch
 
 jobs:
-  build:
-    runs-on: ubuntu-20.04
+  lint:
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-
-      - name: Install node
-        uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
         with:
-          node-version: "14.x"
-          registry-url: "https://registry.npmjs.org"
+          python-version: "3.10"
+      - uses: pre-commit/action@v3.0.0
 
-      - name: Install Python
-        uses: actions/setup-python@v2
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
         with:
-          python-version: "3.x"
-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install twine wheel jupyter-packaging jupyterlab
-
-      - name: Build
-        run: |
-          python setup.py generate_source
-          python setup.py sdist bdist_wheel
-
-      - name: Build
-        run: |
-          cd js
-          npm pack
-
-      - name: Upload builds
-        uses: actions/upload-artifact@v3
+          python-version: "3.10"
+      # https://github.com/webpack/webpack/issues/14532
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: install build
+        run: python -m pip install build
+      - name: build wheel
+        run: python -m build
+      - uses: actions/upload-artifact@v3
         with:
           name: ipyvuetify-dist-${{ github.run_number }}
           path: |
             ./dist
             ./js/*.tgz
 
   test:
-    needs: [build]
-    runs-on: ubuntu-20.04
+    needs: [lint, build]
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, "3.10", "3.11"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
As 3.7 as already reached end of life, keeping the previous one that is already dead seems too much for me. if you would like to keep it please run the test again as I may have introduce some pathlib mechanism that were not available in 3.6

Reply:
solved in https://github.com/widgetti/ipyvuetify/pull/259/commits/f3980bb0045ec474dbe44c581f727d9f1c61ec48

Reply:
it seems pytest-playwright is not compatible with 3.6 either, making the build fail. what should I do ?

Reply:
I think we only need solara[pytest] and thus `${wheel}[test]` in the ui-test job. Removing the `[test]` from `$(find dist -name *.whl)[test]` will fix it.

Reply:
First I would like to solve the 3.6 build. all the others from 3.7 to 3.11 worked but 3.6 failed because playwright is not working. I cannot drop [test] there as it's where the test requirements are but if they are based on playwright it seems my life will become more complicated

Reply:
I think you removed `[test]` from the `ui-test` instead of the `test` job.

Reply:
you were talking about the ui-test so I checked. I cannot remove [test] from the normal test as solara itself is set in it. 

Just to advocate again in favor of dropping python 3.6, many of the top used widget libs that I know (+other packages that are not widgets) have already dropped it:
- ipywidgets: https://pypi.org/project/ipywidgets/ >=3.7
- ipyleaflet: https://pypi.org/project/ipyleaflet/ >= 3.7
- matplotlib: https://pypi.org/project/matplotlib/ >= 3.8
- pandas: https://pypi.org/project/pandas/ >= 3.8

To make the test work on 3.6 we should redesign them without use of playwright which is use directly as a fixture.

Reply:
It's fixed with removing `[test]` in the test job. We can postpone dropping 3.6 support.